### DISCUSSION TITLE
Anpassen von Exporten oder Berichten

### DISCUSSION THREAD

**Discussion by berndz2001** [kimai/kimai#3199] [Questions & Answers]
Hallo zusammen,
ich habe die Diskussionen durchsucht und nichts passendes zu den Anforderungen eines Kunden gefunden, der eventuell auf Kimai2 zukünftig nutzen will.
Sind folgende Anforderungen umsetzbar?
- Auswertung nach Kunde und Projekt und Tätigkeit und Monat
- Zusammenfassung der Stunden pro Leistung pro Monat (also z.B. alle in einem Monat angefallenen Stunden für den Entwurf)
- Kennzeichnung von Stunden, die nicht abgerechnet werden (z.B. Fahrzeiten)
- Kennzeichnung von Stunden, die bereits abgerechnet wurden (nicht  zwingend nötig)
- keine Auflistung der Stunden oder Tage pro Mitarbeiter (es soll für den Kunden nicht sichtbar sein, wer welche Stunden gemacht hat)
- eigenes Logo in Berichte oder Exporte einbinden
Ich bin gespannt auf die Antworten.
Mit freeundlichen Grüßen.
Bernd Zeitzen

**Comment by kevinpapst** [comment]
Ja, ist alles machbar. 
Da die Anforderungen aber nur grob sind, kann ich auch nicht genauer drauf eingehen.
Gerne können wir einen Beratungstermin telefonisch bzw. via Screenshare mit Vorführung vereinbaren.
Dazu einfach mich [per Email kontaktieren](mailto:kevin@kevinpapst.de).

**Reply by berndz2001** [reply]
Danke für die schnelle Antwort. Ich gebe das weiter und melde mich per Mail.
